packages/perseus-editor/src/components/__tests__/locked-function-settings.test.tsx

```diff
@@ -363,6 +373,138 @@ describe("Locked Function Settings", () => {
                     domain: [3, Infinity],
                 });
             });
+
+            test("restriction labels reflect the directional axis specified", async () => {
+                // Arrange
+                render(
+                    <LockedFunctionSettings
+                        {...defaultProps}
+                        directionalAxis="x"
+                        expanded={true}
+                        onChangeProps={onChangeProps}
+                    />,
+                    {wrapper: RenderStateRoot},
+                );
+
+                // Assert - "x" axis means "domain"
+                let minField = screen.getByText("domain min");
+                expect(minField).toBeInTheDocument();
+
+                // Arrange
+                render(
+                    <LockedFunctionSettings
+                        {...defaultProps}
+                        directionalAxis="y"
+                        expanded={true}
+                        onChangeProps={onChangeProps}
+                    />,
+                    {wrapper: RenderStateRoot},
+                );
+
+                // Assert - "y" axis means "range"
```

Comment:
I like your system of adding extra context to the arrange/act/assert comments. I'm gonna start doing this too.